tests/integration/test_http_source_dictionary/test.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
```

Comment:
I think we should place this test into queries/0_stateless and create `.sh` wrapper for it.

Reply:
@FArthur-cmd after discussion it make sense to move it under stateless tests because actually we already have similar tests that start http server and make some checks.